src/app/work-item/work-item-search/work-item-search.component.html

```diff
@@ -1,4 +1,4 @@
-<div id="search-component">
+<div id="workItemSearch_componentContainer">
   <h4>Work Item Search</h4>
   <input #searchBox id="search-box" (keyup)="search(searchBox.value)" />
```

Comment:
@nimishamukherjee in most of the cases we are not sure if they are final designs. for view detail component, board component there will be completely new designs. We can do refactoring of Ids based on your all comments once we got the final designs.